packages/restlessness-core/assets/boilerplate/configs/schedule-events.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Let's choose a single name like endpoints, event is generic, maybe schedules.json or crons.json can work